src/System.Numerics.Vectors/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Matrix3x2</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Matrix4x4</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Plane</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Quaternion</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Vector2</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Vector3</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Vector4</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0003</DiagnosticId>

```

Comment:
https://learn.microsoft.com/en-us/dotnet/fundamentals/apicompat/diagnostic-ids

| CP0003 | Some part of the assembly identity (name, public key token, culture, retargetable attribute, or version) does not match on both sides of the comparison. | Update the assembly identity so that both sides match. |

Seems like one we want to fix.

Reply:
Fixed by setting the key to Microsoft.